src/query/ts/m3db/encoded_block_iterator_builder.go

```diff
@@ -59,10 +61,13 @@ func (b *encodedBlockBuilder) add(
 	str := start.String()
 	if _, found := b.blocksAtTime[str]; !found {
 		// Add a new encoded block
+		// NB: the lookback will always be 0, as splitting series into blocks is not
```

Comment:
In the future, we plan to support this, right?

Reply:
Potentially; it may not be worth dealing with the added complexity in this case.